### PR TITLE
fix: fix buildRoleLinks

### PR DESCRIPTION
**Pull request by nodece**
Fix: https://github.com/casbin/node-casbin/issues/261
Fix: https://github.com/casbin/node-casbin/issues/269

Signed-off-by: Zixuan Liu <nodeces@gmail.com>

### CODE REVIEW

**Comment by hsluoyz** [comment]
@nodece what issue does this resolve?

**Comment by nodece** [comment]
@hsluoyz #261

File: examples/issues/node_casbin_261_model.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[request_definition]
```
Comment (hsluoyz) [inline examples/issues/node_casbin_261_model.conf:1]: `node_casbin_261_model.conf` I don't like the 261 naming. Can you name it in a normal way as other model files? Or do Golang have any such model files to use?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: examples/issues/node_casbin_261_model.conf (hunk 1 above)
Comment (nodece) [reply examples/issues/node_casbin_261_model.conf:1]: do you have the idea of this name?

Comment (hsluoyz) [reply examples/issues/node_casbin_261_model.conf:1]: See Golang: https://github.com/casbin/casbin/search?q=batchEnforce&type=issues

Comment (nodece) [reply examples/issues/node_casbin_261_model.conf:1]: What do you think of using model text and string adapter?

Comment (hsluoyz) [reply examples/issues/node_casbin_261_model.conf:1]: I think if Golang has an similar example model & policy, we should also use the same. Does Go have it?

**Review by Zxilly — APPROVED** [review]

File: examples/issues/node_casbin_261_model.conf (hunk 1 above)
Comment (hsluoyz) [reply examples/issues/node_casbin_261_model.conf:1]: @nodece you can name it from the meaning, use a similar naming way as other files. `node_casbin_261_model.conf` is just a bad name, as it's coming from Node-Casbin GitHub issue #261, this is not good.

Comment (nodece) [reply examples/issues/node_casbin_261_model.conf:1]: `go-casbin` does not have a similar model and policy, `node_casbin_261_model.conf` looks clear, please let me know if you have more good names.

Comment (hsluoyz) [reply examples/issues/node_casbin_261_model.conf:1]: @nodece 

rbac_g2_model.conf

rbac_g2_policy.conf

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**